algorithms/refinement/reflection_manager.py

```diff
@@ -314,14 +319,23 @@ def from_parameters_reflections_experiments(
                 *params.weighting_strategy.constants, stills=do_stills
             )
 
+        # Check for deprecated parameter
+        if params.trim_scan_edges > 0.0:
+            warnings.simplefilter("default")
```

Comment:
this line changes the process-wide warning settings and probably should not be here
```suggestion
```